### PR TITLE
(styles_minimum) Some Improve Styles

### PR DESCRIPTION
**Pull request by crscaballero**
[Asana Task: Style issue in drop-down](https://app.asana.com/0/inbox/1112248602150293/1161291804766377/1161529795761686)

[Asana Task: Style form labels and radio text to match design](https://app.asana.com/0/inbox/1112248602150293/1161678650461184/1161686431972235)

[Asana Task: For presentation to client lets comment out client login and make a payment buttons in pre-header and style the search text so it is open sans 500](https://app.asana.com/0/inbox/1112248602150293/1161718825334569/1161719922624108)
* about the above task, still pending comment the login button, it seems to me that have to be done from admin section (not sure)
* And during the next days will be good add this font to the global header section
`<link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">`
by the moment I've add a font from the same family (approved by Dav), which will be showed until get the right one "[OpenSans](https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans)"

### CODE REVIEW

File: .ddev/.gitignore [hunk 1]
```diff
@@ -3,21 +3,20 @@
```
Comment (abeal-hottomali) [inline .ddev/.gitignore]: @crscaballero , why was this file modified?

Comment (crscaballero) [reply .ddev/.gitignore]: It was a mistake pushed that file, for some reason, when I do the compile/start process, that file it's been modified, so I shouldn't be added to the commit
